### ISSUE TITLE
x/mobile: delete deprecated Callbacks type

### ISSUE THREAD

**Issue by rakyll** [golang/go#10896]
The callback terminology feels odd in Go code. We should rather call them Handlers. I have surveyed godoc to see how community feels and saw only a few instances of the use of callback term. The standard library and the community emerges in the use of handlers. If we keep the callback terminology around, it might give a false impression that they are of another kind of handlers. Let's clean the mobile repo from callbacks and call them handlers.

/cc @crawshaw @hyangah @minux

**Comment by minux** [comment]
SGTM.

**Comment by rakyll** [comment]
Anyone else agrees or disagrees? Go 1.5 will ship a stable app package. If we don't want it to be moved to exp, we either need to make the renaming or close this issue as a WONTFIX.

**Comment by crawshaw** [comment]
I'm still idly thinking about this. Handlers is fine, but I'm not really bothered by Callbacks. Either way, @nigeltao is exploring an events channel and that should happen first.

**Comment by rakyll** [comment]
I'd like to see a design doc for such a significant change.

**Comment by nigeltao** [comment]
Yes, I want to delete Callbacks, not rename them.

I have the code working, but what's taking up my time now is writing the comments, or what would effectively be a design doc. :-)

**Comment by nigeltao** [comment]
I still need to write more commentary or "design" comments, and I still haven't patched all of the examples or e.g. the sprite code, but here's a work in progress: https://go-review.googlesource.com/11351
